cypress/integration/non-regression-tests-suite/event-signup-flow.spec.js

```diff
@@ -565,63 +565,72 @@ describe("Verify page load " + new Date(), () => {
 			});
 	});
 	it('It navigates to selected "Online Events" page', function () {
+		let event_Name;
```

Comment:
Can we keep these consistent with `camelCase`?

Reply:
Resolved